elements/howto-tabs/howto-tabs.js

```diff
@@ -78,12 +96,10 @@
       if (!this.hasAttribute('role'))
         this.setAttribute('role', 'tablist');
 
-      // Currently, `slotchange` does not fire when an element is upgraded. For
-      // this reason, the element always processes the slots after the inner
-      // elements have been defined. If the current behavior of the `slotchange`
-      // event is change (as proposed in
-      // [this issue](https://github.com/whatwg/dom/issues/447)), the code below
-      // can be removed.
+      // Up until recently, `slotchange` events did not fire when an element is
+      // upgraded by the parser. For this reason, the element invokes the
+      // handler manually. Once the new behavior lands in all browsers, the code
+      // below can be removed.
```

Comment:
I removed the link because the spec change has been merged and it doesn’t say anything about the current state of individual browsers, which is probably what most readers would be interested in.